lib/rubocop/cop/style/empty_case_condition.rb

```diff
@@ -70,21 +70,24 @@ def correct_case_whens(corrector, case_node, when_nodes)
         end
 
         # Since an if condition containing commas is not syntactically valid, we
-        # correct `when x,y` to `if [x,y].any?`.
+        # correct `when x, y` to `if x || y`.
         def correct_multiple_alternative_whens(corrector, when_nodes)
           when_nodes.each do |when_node|
-            children = when_node.children
+            *children, _ = when_node.children
 
             # In `when a; r` we have two children: [a, r].
             # In `when a, b, c; r` we have 4.
-            next unless children.size > 2
+            # In `when a, b then r` we have 3.
```

Comment:
I think this comment should be moved above the `*children, _` line, else it's a bit confusing because `r` wouldn't be in `children` (since it's captured by the `_`).